Cohen-Koen.md

```diff
@@ -0,0 +1,32 @@
+﻿---
+name: "@Cohen-Koen"
+project: "Screens for Cougar MFD"
+---
+
+#  Screens for Cougar MFD
+
+##  Summary
+
+I am going to build and integrate screens with the Thrustmaster Cougar MFD, this will allow for a realistic look on MFD, I wish to make an affordable option for anyone who works with flight simulators like me. There are alternatives however I wish to make them well integrated and something that people can use as a guide for their own projects
+
+##  Plan
+
```

Comment:
What will the mount be made of?

Reply:
at first it should be made out of wood or something that I can work with given the time, however I have access to 3d printers as a part of my stem club, so once I reach that point I will design a mount with 3DS max and I will put the 3d printed pieces wherever people want. I would be learning a bit about 3d printing as its not my main workplace, definitely a skill I should learn. So short answer, wood while I work, then a 3d model once I have the wooden bit ready, should just clip on or something easy and cheap.